client/package.json

```diff
@@ -33,6 +33,8 @@
     "build:dev:client": "webpack -w --config webpack.client.rails.build.config.js",
     "build:dev:server": "webpack -w --config webpack.server.rails.build.config.js",
     "build:server": "webpack --config webpack.server.rails.build.config.js",
+    "build:test": "npm run build:client && npm run build:server",
+    "build:production": "npm run build:production:client && npm run build:production:server",
```

Comment:
Be sure to create 2 npm tasks like this. The first one, `build:test` is for building the assets for running tests. The second one `build:production` if for building assets before the precompile step.